src/robocop/linter/runner.py

```diff
@@ -189,7 +232,9 @@ def make_reports(self) -> None:
         for report in self.reports.values():
             prev_result = prev_results.get(report.name) if prev_results is not None else None
             report.generate_report(
-                diagnostics=diagnostics, config_manager=self.config_manager, prev_results=prev_result
+                diagnostics=diagnostics,
```

Comment:
I saw several lines split, did you perhaps run it with formatter with configured length 88 on it? We use 120 by default so it shouldn't split, unless I'm not seeing something.